website/docs/docs/collaborate/model-performance.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Model performance"
+sidebar_label: "Model performance"
+description: "Learn about ."
+---
+
+dbt Explorer provides metadata on dbt Cloud runs for in-depth model performance and quality analysis. This feature assists in reducing infrastructure costs and saving time for data teams by highlighting where to fine-tune projects and deployments &mdash; such as model refactoring or job configuration adjustments.
+
+<LoomVideo id='98f33b3b7a374df0b7c04747eae6ef44' />
+
+:::tip Public preview 
+
+The model performance feature is now available in dbt Explorer! Check it out! 
```

Comment:
The second sentence might not be easily understood when English is someone's second language. Definitely bring on the excitement. Maybe "Go explore!" (ha ha)